src/main/java/pinger/Pinger.java

```diff
@@ -26,6 +26,9 @@ public static void main(String[] args) throws IOException {
                 addField("Header Fields", conn.getHeaderFields());
                 addField("Content", content);
             }
+        } catch(Exception e) {
+            System.out.println("error occured);
```

Comment:
This doesn't compile
You can test your changes by running `./gradlew run`, or `gradlew.bat run`(windows)